index.js

```diff
@@ -1,6 +1,41 @@
 /* jshint node: true */
 'use strict';
 
+var path = require('path');
+var mergeTrees = require('broccoli-merge-trees');
+
 module.exports = {
-  name: 'ember-redux'
+  name: 'ember-redux',
+
+  included: function(app) {
+    // Import UMD redux and redux-thunk
```

Comment:
It is possible to check for ember-cli >= 2.9 here and warn appropriately. 

Also it may be a good idea to warn if browserify is installed. While there are other valid uses for it, Im worried about someone importing nmp:redux from their code resulting in two versions of redux (commonjs+umd) in vendor.js